modules/nlu/src/backend/predict-pipeline.ts

```diff
@@ -237,12 +237,15 @@ async function predictIntent(input: PredictStep, predictors: Predictors): Promis
   const ctxToPredict = input.ctx_predictions.map(p => p.label)
   const predictions = (
     await Promise.map(ctxToPredict, async ctx => {
+      let preds: sdk.MLToolkit.SVM.Prediction[] = []
+
       const predictor = predictors.intent_classifier_per_ctx[ctx]
-      if (!predictor) {
-        return
+      if (predictor) {
+        const features = [...input.utterance.sentenceEmbedding, input.utterance.tokens.length]
```

Comment:
ins't there a function for this ? or it still hasn't been merged ?  gentIntentFeatures or something like that. 

if it doesn't exist, means that it hasn't been merge, can you leave a todo comment just to make sure we don't loose this one

Reply:
sure!